.github/workflows/build.yaml

```diff
@@ -122,11 +122,10 @@ jobs:
           cmake "${{github.workspace}}/qpid-proton" \
             "-DCMAKE_INSTALL_PREFIX=${InstallPrefix}" \
             "-DCMAKE_BUILD_TYPE=${BuildType}" \
-            "-GNinja" \
             ${ProtonCMakeExtraArgs}
 
       - name: qpid-proton cmake build/install
-        run: cmake --build "${ProtonBuildDir}" --config ${BuildType} -t install --parallel 6
```

Comment:
if we leave out the --parallel command line option, is it the same as --parallel 1 ?

Reply:
Depends on the generator. For `make`, yes, for `ninja`, no. I think it is better to be explicit.